store/seed/release/10000__reset.sql

```diff
@@ -0,0 +1,118 @@
+-- For testing, we reset data on each run
+-- Do not reset bb.auth.secret so that we don't need to re-login after restart
```

Comment:
This is used for demo+release binary (demo.bytebase.com, staging.bytebase.com), so it's the same w or w/o it.

Reply:
This is really a fat foot and we need to design a better scheme for it. Otherwise, people who first see it will likely think it's resetting the release data and it may even be misused.

Reply:
Yeah, how about renaming it to "demo" instead of "seed"?

Reply:
SGTM

Reply:
Done.